packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1582,6 +1582,26 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   static RenderObject? get debugActiveLayout => _debugActiveLayout;
   static RenderObject? _debugActiveLayout;
 
+  /// Set [debugActiveLayout] to null when [inner] callback is called.
```

Comment:
Isn't this change the same as https://github.com/flutter/flutter/pull/114003?

Reply:
Almost same (as mentioned there), but with one more thing:

This PR makes this function *public* instead of *private*. Then, users of the secondary pipeline owners can call that function. If only have that PR, then cannot.